src/main/java/ai/elimu/tasks/LetterToAllophoneMappingUsageCountScheduler.java

```diff
@@ -50,7 +50,6 @@ public synchronized void execute() {
         // Update the values previously stored in the database
         for (LetterToAllophoneMapping letterToAllophoneMapping : letterToAllophoneMappingDao.readAll()) {
             logger.info("letterToAllophoneMapping.getId(): " + letterToAllophoneMapping.getId());
-            logger.info("letterToAllophoneMapping.getLetter().getText(): " + letterToAllophoneMapping.getLetter().getText());
```

Comment:
Instead of removing this altogether, can we log the list of letters here:
`logger.info("letterToAllophoneMapping.getLetters():" + letterToAllophoneMapping.getLetters().stream().map(Letter::getText).collect(Collectors.joining()));`

Reply:
Similarly if required- https://github.com/elimu-ai/webapp/pull/1085/files#diff-2a44ad68a27b9dce2f0008f4aa0bd40df41827349e6a32fd391cb7d24ccd11c3R25

Reply:
@Keerthi4308 Wow, I was looking for a way to solve this, but I did not know how to do it. Thank you very much 😁

I've updated the pull request with your suggestion.